### ISSUE TITLE
(virtualbox) - Params are not respected on choco upgrade all

### ISSUE THREAD

**Issue by noaho** [chocolatey/chocolatey-coreteampackages#1211]
## Expected Behavior
If I specify /NoDesktopShortcut /NoExtensionPack, it will stay that way. 

## Current Behavior
/NoDesktopShortcut /NoExtensionPack work until the next package upgrade / choco upgrade, then the extension pack is back and a desktop shortcut is created. 

## Steps to Reproduce (for bugs)
1. choco install -y virtualbox --version "6.0" --params "/NoDesktopShortcut /NoExtensionPack"
2. choco upgrade all -y

**Comment by NoahO** [comment]
Thanks!

**Comment by stumbaumr** [comment]
This seems not to work as described
```
C:\WINDOWS\system32>choco feature list
Chocolatey v0.11.2
[x] checksumFiles - Checksum files when pulled in from internet (based on package).
[x] autoUninstaller - Uninstall from programs and features without requiring an explicit uninstall script.
...
[x] useRememberedArgumentsForUpgrades - Use Remembered Arguments For Upgrades - When running upgrades, use arguments for upgrade that were used for installation ('remembered'). This is helpful when running upgrade for all packages. Available in 0.10.4+. This is considered in preview for 0.10.4 and will be flipped to on by default in a future release.
[ ] ignoreUnfoundPackagesOnUpgradeOutdated - Ignore Unfound Packages On Upgrade Outdated - When checking outdated or upgrades, if a package is not found against sources specified, don't report the package at all. Available in 0.10.9+.
...
[x] usePackageRepositoryOptimizations - Use Package Repository Optimizations - Turn on optimizations for reducing bandwidth with repository queries during package install/upgrade/outdated operations. Should generally be left enabled, unless a repository needs to support older methods of query. When disabled, this makes queries similar to the way they were done in Chocolatey v0.10.11 and before. Available in 0.10.14+.

C:\WINDOWS\system32>choco list --local zabbix-agent
Chocolatey v0.11.2
zabbix-agent.install 5.2.7
1 packages installed.

C:\WINDOWS\system32>
```

I install zabbix-agent.install with this PowerShell:
```
$myFQDN = [System.Net.Dns]::GetHostByName($env:computerName).Hostname.ToLower()
$Software = @()
$Software += [pscustomobject]@{name = "tortoisesvn"; params = ""; post_inst = ""}
...
$Software += [pscustomobject]@{name = "zabbix-agent.install"; version = "5.2.7"; params = "/LOGTYPE:system /ENABLEREMOTECOMMANDS:1 /SERVER:zproxy02,zabbix /SERVERACTIVE:zproxy02 /HOSTNAME:$($myFQDN)"; post_inst = "C:\monitoring\files\windows\scripts\UpdateZabbixConf.ps1"}

Set-MpPreference -EnableControlledFolderAccess AuditMode

foreach ($item in $Software){
   & "$($env:ChocolateyInstall)\bin\choco" install -y --allow-downgrade $($item.name) --version="$($item.version)" --params="$($item.params)"
   If ($item.post_inst) {
		& "$($item.post_inst)"
   }
   remove-item -path "$($env:PUBLIC)\Desktop\*.lnk" -force
}
Set-MpPreference -EnableControlledFolderAccess Enabled
```

When running choco upgrade -y all it will upgrade the Zabbix Agent to 6.0.4...

How can I keep that version 5.2.7 and upgrade all the rest?
